components/layout_2020/style_ext.rs

```diff
@@ -249,6 +249,7 @@ pub(crate) trait ComputedValuesExt {
         LogicalVec2<Size<Au>>,
         LogicalVec2<Size<Au>>,
         PaddingBorderMargin,
+        bool, /* has_block_percentage */
```

Comment:
clippy is really going to complain about this. Do you mind returning a new data structure. I'm not too pick about the name, but it would be nice to avoid the new clippy warning. Or perhaps this is being addressed in another PR?